### ISSUE TITLE
Wrong file/version for farms

### ISSUE THREAD

**Issue by proxymus** [turnermm/ckgdoku#25]
I've tried to update my ckgdoku farm plugin, but the link [cked_farm_4.7.zip](https://github.com/turnermm/ckgdoku/archive/cked_farm_4.7.zip) installs the ckededit version, what I've noticed to late. After this I had both versions installed (ckgedit_farm and ckgdoku_farm). Dokuwiki prompts me to disable one of them.
Is this the expected behaviour?

Thank you in advance.

**Comment by turnermm** [comment]
Sorry about that.  It's been corrected:   https://github.com/turnermm/ckgdoku/archive/cked_farm_4.7.zip.  And, yes, you can't run both ckgedit and ckgdoku at the same time,  They interfere with one another.

**Comment by psy-q** [comment]
The farm version appears to generate a notice that a newer version is available (2017-10-26) but then points at the [non-farm ckgdoku](https://github.com/turnermm/ckgdoku/archive/master.zip) repository for downloading.

Is it necessary to keep these things apart, couldn't the normal version of the plugin detect whether it's in a farm enviornment and adjust accordingly?

**Comment by turnermm** [comment]
This is just a book-keeping mismatch.  The Extension manager will always point to the ckgdoku master distribution, because ckgdoku has only one entry in the dokuwiki plugins listings.

**Comment by turnermm** [comment]
>Is  it necessary to keep these things apart, couldn't the normal version of  the plugin detect whether it's 
 in a farm enviornment and adjust  accordingly?

I have been working on this and the following distribution is prepared to be merged into the master, so that from now on there will be only one ckgdoku:
https://github.com/turnermm/ckgdoku/archive/jq_3.0.zip
I've tested it myself, but before finalizing it, it would be helpful if someone else would try it out.  Thanks.

**Comment by proxymus** [comment]
I will give it a try today.

**Comment by turnermm** [comment]
Thanks.

**Comment by proxymus** [comment]
The update was working without any problems. I've tested several editor functions and haven't noticed any problems so far. Is there anything special what I should test?

**Comment by turnermm** [comment]
No,  it's just reassuring to know that is is working without any problems. I'm always switching back and forth between ckgdoku and ckgedit and don't use either with the same intensity as someone who needs dokuwiki on a daily basis or uses it in a multi-user environment.  I made this version the download distribution for farms, and in a few weeks I'll make it the  dokuwiki download distribution for both farms and non-farms.  I've also  combined farms/non-farms  in ckgedit and there I made the new combined farms/non-farms version the dokuwiki download distribution.   Since ckgedit is more widely used, if any problems show up I'll hear about it soon.   Thanks for testing.
